File: week6/윤여준/예상 대진표.cpp
#include <bits/stdc++.h>
using namespace std;

int solution(int n, int a, int b) {
    int answer = 1;
    
    while((a + 1) / 2 != (b + 1) / 2) {
        if((a + 1) / 2 != 1) a = (a + 1) / 2;
        if((b + 1) / 2 != 1) b = (b + 1) / 2;
        answer++;
    }
    
    return answer;
}

File: week6/박현국/예상대진표.java
class Solution
{
    public int solution(int n, int a, int b)
    {
        int answer = 0;
        a--;
        b--;

        while (a != b) {
            a /= 2;
            b /= 2;
            answer++;
        }

        // INDEX / 2 + 1
        return answer;
    }
}